next-tavla/src/Board/scenarios/Board/index.tsx

```diff
@@ -19,14 +18,14 @@ function BoardTile({ tileSpec }: { tileSpec: TTile }) {
 function Board({ board, style }: { board: TBoard; style?: CSSProperties }) {
     if (!board.tiles || !board.tiles.length)
         return (
-            <Tile className={classes.emptyTile}>
+            <Tile className="flex items-center justify-center">
                 <p>Du har ikke lagt til noen stoppesteder enda.</p>
             </Tile>
         )
 
     return (
         <div
-            className={`${classes.board} ${getFontScale(
```

Comment:
Kan vi bruke noen av de `em`-variablene i tailwind-configen her i stedet for å hardkode `[10px]`?